INSTALL.md

```diff
@@ -23,32 +23,54 @@ To build **pokeruby.gba**:
 	make -j4
 
 
-# Mac
-
-Install [**devkitARM**](http://devkitpro.org/wiki/Getting_Started/devkitARM).
-
-Then in **Terminal**, run the following commands.
-
-	xcode-select --install
-
-	export DEVKITPRO=${HOME}/devkitPro
-	echo "export DEVKITPRO=${DEVKITPRO}" >> ~/.bashrc
-	export DEVKITARM=${DEVKITPRO}/devkitARM
-	echo "export DEVKITARM=${DEVKITARM}" >> ~/.bashrc
-
-	git clone https://github.com/pret/pokeruby
-	git clone https://github.com/pret/agbcc
-
-	cd agbcc
-	./build.sh
-	./install.sh ../pokeruby
-
-	cd ../pokeruby
-
-To build **pokeruby.gba**:
-
-	make -j4
-
+# macOS
+
+Installing pokeruby on a Mac requires macOS >= 10.12 (Sierra or higher).
+
+To get pokeruby up and running, run the following commands in Terminal:
+
+Make sure you have the right compilers.
+```
+xcode-select --install
+```
+
+Having [`homebrew`](https://brew.sh/) will make installing other dependencies a lot easier.
+Visit the website for install instructions.
+
+```
+brew install gcc
+brew install md5sha1sum
+```
+   
+Download the [devkitPRO pacman](https://github.com/devkitPro/pacman/releases/tag/v1.0.0) package (`.pkg`) and install using the native Mac Installer.
```

Comment:
Remove the `/tag/v1.0.0` so as to link to whatever the latest version of dkp-pacman happens to be (1.0.1 at present time)

Reply:
1.0.1 didn't work for me last time I tried